tsc/OpenTimelineIO_TSC_Charter.md

```diff
@@ -0,0 +1,67 @@
+# Technical Charter (the "Charter") for OpenTimelineIO a Series of LF Projects, LLC
+
+This charter (the "Charter") sets forth the responsibilities and procedures for technical contribution to, and oversight of, the OpenTimelineIO project, which has been established as OpenTimelineIO a Series of LF Projects, LLC (the "Project").  LF Projects, LLC ("LF Projects") is a Delaware series limited liability company. All Contributors to the Project must comply with the terms of this Charter.
```

Comment:
My confusion around the charter would be resolved by declaring the relationship between the ASWF and the LF for the purposes of the charter. For example, 

ASWF ("The Academy Software Foundation") is a group of projects within the LF Projects.